lib/tpm2_alg_util.c

```diff
@@ -351,7 +351,7 @@ UINT8* tpm2_extract_plain_signature(UINT16 *size, TPMT_SIGNATURE *signature) {
         if (!buffer) {
             goto nomem;
         }
-        memcpy(buffer, &hmac_sig, *size);
```

Comment:
I would also have this at the beginning of the set since is a trivial fix (although maybe you did since as mentioned Github seems to mess with the commits order).